code/game/machinery/dna_infuser/organ_sets/goliath_organs.dm

```diff
@@ -0,0 +1,177 @@
+
+#define GOLIATH_ORGAN_COLOR "#875652"
+#define GOLIATH_SCLERA_COLOR "#ac0f32"
+#define GOLIATH_PUPIL_COLOR "#FF0000"
+
+#define GOLIATH_COLORS GOLIATH_ORGAN_COLOR + GOLIATH_SCLERA_COLOR + GOLIATH_PUPIL_COLOR
+
+///bonus of the goliath: you can swim through space!
+/datum/status_effect/organ_set_bonus/goliath
+	organs_needed = 4
+	bonus_activate_text = span_notice("goliath DNA is deeply infused with you! You can now endure walking on lava!")
+	bonus_deactivate_text = span_notice("Your DNA is once again mostly yours, and so fades your ability to survive on lava...")
```

Comment:
Would prefer a better description for the first part, "Your DNA is once again mostly yours". I set a bad pattern by making them generic, would prefer if they all had a unique DNA loss message for the first half.